style.css

```diff
@@ -115,26 +118,69 @@ html, body {
 .header__link {
     text-transform: uppercase;
     font-size: 12px;
+    letter-spacing: 1px;
     text-decoration: none;
-    color: #000000;
+    color: var(--main-text-color);
     font-family: 'Montserrat', sans-serif;
-    font-weight: bold;
+    cursor: pointer;
+}
+.header__link:hover, .header__cartlink:hover, .header__sublink:hover {
+    color: #a0522d;
 }
 .header__cartlink {
     font-size: 12px;
-    text-decoration: none;
-    color: #000000;
+    text-decoration: none;  
+    color: var(--main-text-color);
     font-family: 'Poppins', sans-serif;
     font-weight: bold;
     text-align: center;
- }
+}
+.form-search {
+    display: block;
+}
+.form-search input {
+    all: unset;
```

Comment:
what is `all`?

Reply:
to remove default field styling for form-search